### PR TITLE
More API Coverage

### PR DESCRIPTION
**Pull request by erasche**
As I'm playing more with parsec, I'm finding some other features not well covered so here's a PR with them. It is not a coherent collection of stuff, my apologies.

- get_users now supports server side filtering that's available
- there's a function to get a user's API key without resetting it.
- /api/version

### CODE REVIEW

**Comment by bgruening** [comment]
More!!! :)

**Comment by martenson** [comment]
And faster! 💯

File: bioblend/galaxy/users/__init__.py [hunk 1]
```diff
@@ -145,3 +167,21 @@ def delete_user(self, user_id, purge=False):
...
+        :param user_id: encoded user ID
+
+        :rtype: str
+        :return: the API key for the user
+        """
+
+        url = self.gi._make_url(self, None)
+        url = '/'.join([url, user_id, 'api_key', 'inputs'])
+        payload = {}
+        payload['user_id'] = user_id
```
Comment (nsoranzo) [inline bioblend/galaxy/users/__init__.py]: @erasche I don't think the `payload` is needed here, I'm going to remove it. Let me know if I'm missing something.

**Comment by bgruening** [comment]
I restarted the tests, now everything passes.

**Comment by bgruening** [comment]
upps, this was sended way to late ...

File: bioblend/galaxy/users/__init__.py (hunk 1 above)
Comment (erasche) [inline bioblend/galaxy/users/__init__.py]: you're right, that shouldn't be needed.

**Comment by erasche** [comment]
@nsoranzo thanks for tracking down and documenting all of the specific versions where these features were added. I'll try and be better about that going forward.

**Pull request merged**